src/AccessibilityInsights.Extensions.Telemetry/AITelemetry.cs

```diff
@@ -81,11 +80,11 @@ private static void MergeIntoProperties(EventTelemetry aiEvent, IReadOnlyDiction
         }
 
         /// <summary>
-        /// Process an event asynchronously
+        /// Process an event
         /// </summary>
-        private async void ProcessEvent(EventTelemetry aiEvent)
+        private void ProcessEvent(EventTelemetry aiEvent)
         {
-            await Task.Run(new Action(() => TClient?.TrackEvent(aiEvent))).ConfigureAwait(false);
```

Comment:
The async behavior has been moved into the TelemetryClientWrapper to simplify testing